addon/mixins/spread.js

```diff
@@ -108,9 +108,11 @@ export default Mixin.create({
    * @param {object} spreadableHash - the hash to spread
    */
   _defineSpreadProperties (spreadProperty, spreadableHash) {
+    const supportedSpreadableTypes = A(['instance', 'object'])
+
     assert(
       `${spreadProperty} requires an Ember object or primitive object`,
-      ['instance', 'object'].includes(typeOf(spreadableHash))
+      supportedSpreadableTypes.includes(typeOf(spreadableHash))
```

Comment:
This is exactly the kind of thing I'm talking about. I don't think needing to use Ember array's everywhere that we want to use `Array.prototype.includes` just so that phantomjs will be happy is a good trade-off, unless supporting phantomjs gives us some benefit I'm not aware of.